theia-cpp-electron/README.md

```diff
@@ -1,30 +1,30 @@
-## Theia C/C++ Electron Example
+# Theia C/C++ Electron Example
 
-Build an Electron C/C++ Theia IDE, from the Eclipse Theia platform, using the `package.json` present in this folder. Electron applications are a good
-choice for local development.
+One can uild an Electron (desktop) Theia example application for C/C++ development, using the `package.json` present in this folder. Electron applications are a good choice for local development but not good for Cloud.
 
-### Install dependencies
+## Install dependencies
+
+### 1. Theia development dependencies
 
-#### 1. Theia development dependencies
 See [here](https://github.com/theia-ide/theia/blob/master/doc/Developing.md#prerequisites) for dependencies you'll need to build a Theia application.
 
-#### 2. clangd
+### 2. clangd
+
 You will need to install the [clangd](https://clang.llvm.org/extra/clangd/Installation.html) language server.
 
-By default, the `clangd` executable is assumed to be named the same and be in the system's path. You can configure an alternate name using the preference `cpp.clangdExecutable`.
+By default, the `clangd` executable is assumed to be named the same and be in the system's path. You can configure an alternate name using the preference `clangd.path`.
 
-If you wish to provide startup arguments to `clangd`, use the preference `cpp.clangdArgs`.
+If you wish to provide startup arguments to `clangd`, use the preference `clangd.arguments`.
 
-You may override the default value of preferences by editing the `preferences` entry in `package.json`.
+## Configurations
 
-### Configurations
+### clang-tidy integration
 
-#### clang-tidy integration
-If you wish to use the [clang-tidy linter integration](https://github.com/theia-ide/theia/blob/master/packages/cpp/README.md#using-the-clang-tidy-linter), you will need to install clangd version 9 or higher, that has a built-in `clang-tidy`, that can provide diagnostics in your Theia editor as you type. You also need to enable the integration using the `cpp.clangTidy` preference.
+If you wish to use the _clang-tidy linter integration_ you will need to install clangd version 9 or higher, that has a built-in `clang-tidy`, that can provide diagnostics in your Theia editor as you type.
```

Comment:
minor: the wording sounds a bit off here. I was thinking something along the lines as:

"If you wish to use the _clang_tidy linter integration_ you will need to install clangd version 9 or higher which has a built-in `clang-tidy` available. This will provide diagnostics in your Theia editor as you type."